chirp/drivers/ft1d.py

```diff
@@ -43,18 +44,41 @@
   } message;
 } opening_message;
 
-#seekto 0x049a;
+// Settings
+#seekto 0x049A;
 struct {
   u8 vfo_a;
   u8 vfo_b;
 } squelch;
 
-#seekto 0x04c1;
+// GM / Digital
+#seekto 0x04BA;
 struct {
-  u8 beep;
-} beep_select;
+    u8 unknown:3,
+        scan_resume:5;          // 52 SCN.RSM
+    u8 unknown1:3,
+       dw_resume_interval:5;       // 22 DW RSM
+    u8 unknown2;
+    u8 unknown3:3,
+        apo:5;                  // 02 APO
+    u8 unknown4:6,
+        gm_ring:2;              // 24 GM RNG
+    u8 temp_cf;               // Placeholder as not found
+    } first_settings;
+
+// GM / Digital
+// already at 0x04c0;
+// Caution: overloaded at 04c1! Code change necessary
```

Comment:
Is the "code change necessary" the one you're making in this commit, or are you saying *another* change is needed that this does not account for?